tests/Jenkins/dirac-cfg-update-server.py

```diff
@@ -405,6 +405,7 @@
 csAPI.setOption("Operations/Defaults/ResourceStatus/Policies/AlwaysBannedForSite/policyType", "AlwaysBanned")
 csAPI.setOption("Operations/Defaults/ResourceStatus/Policies/AlwaysBannedForSite/matchParams/element", "Site")
 
+csAPI.setOption("Operations/Defaults/EnableActivityMonitoring", "True")
```

Comment:
I think we can fully remove this flag. Instead, always enable the activity monitoring if there's a ES configured.